qpc/messages.py

```diff
@@ -194,9 +199,9 @@
 REPORT_SJ_DOES_NOT_EXIST = \
     'Scan Job %s does not exist.'
 REPORT_SJS_DO_NOT_EXIST = 'The following scan jobs do not exist: %s.'
-REPORT_NO_SUMMARY_REPORT_FOR_SJ = \
+REPORT_NO_DEPLOYMENTS_REPORT_FOR_SJ = \
```

Comment:
Not a blocker but could we possibly combine these two error messages and just add the "scan job/report job" when we are calling the message?